packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayout.tsx

```diff
@@ -13,7 +13,7 @@ require('style-loader!css-loader!react-resizable/css/styles.css');
 require('style-loader!css-loader!./DashboardGridLayout.css');
```

Comment:
As is found in another issue, we should get rid of these loader syntax in our codebase.